exasol_integration_test_docker_environment/lib/test_environment/setup_external_database_host.py

```diff
@@ -5,30 +5,44 @@
 
 import luigi
 
-from exasol_integration_test_docker_environment.lib.base.dependency_logger_base_task import DependencyLoggerBaseTask
-from exasol_integration_test_docker_environment.lib.base.json_pickle_parameter import JsonPickleParameter
-from exasol_integration_test_docker_environment.lib.data.database_credentials import DatabaseCredentialsParameter
-from exasol_integration_test_docker_environment.lib.data.database_info import DatabaseInfo
-from exasol_integration_test_docker_environment.lib.data.docker_network_info import DockerNetworkInfo
-from exasol_integration_test_docker_environment \
-    .lib.test_environment.parameter.external_test_environment_parameter import \
-    ExternalDatabaseXMLRPCParameter, ExternalDatabaseHostParameter
-from exasol_integration_test_docker_environment \
-    .lib.test_environment.ports import Ports
+from exasol_integration_test_docker_environment.lib.base.dependency_logger_base_task import (
+    DependencyLoggerBaseTask,
+)
+from exasol_integration_test_docker_environment.lib.base.json_pickle_parameter import (
+    JsonPickleParameter,
+)
+from exasol_integration_test_docker_environment.lib.data.database_credentials import (
+    DatabaseCredentialsParameter,
+)
+from exasol_integration_test_docker_environment.lib.data.database_info import (
+    DatabaseInfo,
+)
+from exasol_integration_test_docker_environment.lib.data.docker_network_info import (
+    DockerNetworkInfo,
+)
+from exasol_integration_test_docker_environment.lib.test_environment.parameter.external_test_environment_parameter import (
+    ExternalDatabaseHostParameter,
+    ExternalDatabaseXMLRPCParameter,
+)
+from exasol_integration_test_docker_environment.lib.test_environment.ports import Ports
 
 
-class SetupExternalDatabaseHost(DependencyLoggerBaseTask,
-                                ExternalDatabaseXMLRPCParameter,
-                                ExternalDatabaseHostParameter,
-                                DatabaseCredentialsParameter):
-    environment_name : str = luigi.Parameter() # type: ignore
-    network_info : DockerNetworkInfo = JsonPickleParameter(DockerNetworkInfo, significant=False)  # type: ignore
-    attempt : int = luigi.IntParameter(1)  # type: ignore
+class SetupExternalDatabaseHost(
+    DependencyLoggerBaseTask,
+    ExternalDatabaseXMLRPCParameter,
+    ExternalDatabaseHostParameter,
+    DatabaseCredentialsParameter,
+):
+    environment_name: str = luigi.Parameter()  # type: ignore
+    network_info: DockerNetworkInfo = JsonPickleParameter(DockerNetworkInfo, significant=False)  # type: ignore
+    attempt: int = luigi.IntParameter(1)  # type: ignore
 
     def run_task(self):
         database_host = self.external_exasol_db_host
-        if self.external_exasol_db_host == "localhost" or \
-                self.external_exasol_db_host == "127.0.01":
+        if (
+            self.external_exasol_db_host == "localhost"
+            or self.external_exasol_db_host == "127.0.01"
```

Comment:
This looks wrong!
@tkilias fix in this PR or another one?

Reply:
yes, please make it its own 
PR and ticket, because this actually a bug

Reply:
Created #432